tests/ludwig/encoders/test_text_encoders.py

```diff
@@ -158,6 +158,47 @@ def test_hf_ludwig_model_reduce_options(tmpdir, csv_filename, encoder_name, redu
         model.train(dataset=rel_path, output_directory=tmpdir)
 
 
+@pytest.mark.parametrize(
+    "pretrained_model_name_or_path",
+    [
+        "hf-internal-testing/tiny-random-bloom",
+        "hf-internal-testing/tiny-random-OPTModel",
+        "hf-internal-testing/tiny-random-GPTJModel",
+    ],
+)
+def test_hf_ludwig_model_auto_transformers(tmpdir, csv_filename, pretrained_model_name_or_path):
+    input_features = [
```

Comment:
nit: Could you add a short explanation of the motivation behind this test?

Reply:
Added comment.